scripts/azureml-assets/azureml/assets/release/test_results_analysis.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+"""Python scripts for comparing test results with next stage config."""
+from pathlib import Path
+import yaml
+from azureml.assets.util import logger
+import argparse
+import azureml.assets as assets
+import azureml.assets.util as util
+SUPPORTED_ASSET_TYPES = [assets.AssetType.COMPONENT.value]
+
+
+def test_results_analysis(config_file: Path, results_file: Path, asset_dir: Path):
+    """Compare test results with create list."""
+    valid_assets = []
+    with open(results_file) as fp:
+        valid_assets = yaml.load(fp, Loader=yaml.FullLoader)
+
+    covered_assets = []
+    uncovered_assets = []
+    with open(config_file) as fp:
+        config = yaml.load(fp, Loader=yaml.FullLoader)
+        create_list = config.get('create', {})
+        for assets_type in create_list.keys():
+            if assets_type not in SUPPORTED_ASSET_TYPES:
+                continue
+            assets_list = create_list[assets_type]
+            if len(assets_list) == 1 and assets_list[0] == '*':
+                assets_set = util.find_assets(
+                    input_dirs=asset_dir / assets_type,
+                    asset_config_filename=assets.DEFAULT_ASSET_FILENAME)
+                assets_list = [asset.name for asset in assets_set]
+                logger.print(f"find all {assets_type}s: {assets_list}")
+            for asset in assets_list:
+                if asset in valid_assets:
+                    covered_assets.append(asset)
+                else:
+                    uncovered_assets.append(asset)
+
+    logger.print(f"covered assets: {covered_assets}")
+    logger.print(f"uncovered assets: {uncovered_assets}")
```

Comment:
Consdier removing this line because `uncovered_assets` is either empty here, or this line duplicates the information on lines 43-45.